VocalTractLabApi.h

```diff
@@ -383,6 +383,26 @@ C_EXPORT int vtlGesturalScoreToTractSequence(const char* gesFileName,
   const char* tractSequenceFileName);
 
 
+// ****************************************************************************
+// This function gets the duration from a gestural score.
+// Parameters:
+// o gesFileName (in): Name of the gestural score file to synthesize.
+// o audioFileDuration (out): The number of audio samples, the audio file would
+//   have, if the gestural score was synthesized. If not wanted, set to NULL.
```

Comment:
[...] synthesized. This number can be slightly larger than the length of the gestural score because the audio is synthesized in chunks of a constant size.

Reply:
Done.